tests/transformers/test_transformers_model_export.py

```diff
@@ -852,6 +852,9 @@ def test_transformers_pt_model_save_dependencies_without_accelerate(
     assert "torch" in pip_requirements
 
 
+@pytest.mark.skipif(
+    importlib.util.find_spec("accelerate") is not None, reason="fails when accelerate is installed"
```

Comment:
This test is fragile when accelerate is installed (which includes Pytorch as dependency). This is most likely because we test Pytorch/Tensorflow cases one after another in the single environment. Our dependency inference logic run subprocesses and updates environment variable there, which might confuses Transformers to decide which engine to use. 

The similar test above `test_transformers_tf_model_save_without_conda_env_uses_default_env_with_expected_dependencies` also has this skip mark so should be ok.